.github/workflows/audit.yml

```diff
@@ -0,0 +1,34 @@
+name: Security audit
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    paths:
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  push:
+    branches:
+    - main
+
+env:
+  RUST_BACKTRACE: 1
+  CARGO_TERM_COLOR: always
+  CLICOLOR: 1
```

Comment:
Wonder why we need backtraces and color?

Reply:
fixed, was copy/paste from  my standard project boilerplate.

(I tend to use formatted logs and prefer colored output, others on the team tend to use raw logs which is harder with colors)